Library/Homebrew/cmd/shellenv.sh

```diff
@@ -0,0 +1,10 @@
+#:  * `shellenv`:
+#:    Prints export statements - run them in a shell and this installation of
+#:    Homebrew will be included into your PATH, MANPATH, and INFOPATH.
+#:    Tip: have your dotfiles eval the output of this command
+
+homebrew-shellenv() {
+  echo "export PATH=\"$HOMEBREW_PREFIX/bin:#{HOMEBREW_PREFIX}/sbin:\$PATH\""
+  echo "export MANPATH=\"$HOMEBREW_PREFIX/share/man:\$MANPATH\""
```

Comment:
Using single quotes inside double quotes won’t prevent variable expansion but will avoid the need for escaping the double quotes. For example, this line would become `echo "export MANPATH='$HOMEBREW_PREFIX/share/man:\$MANPATH'"` which I find marginally clearer.